pyproject.toml

```
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["flit"]
-build-backend = "flit.buildapi"
+requires = ["flit_core"]
```

```suggestion
requires = ["flit_core >=2,<4"]
```

I recommend people set a maximum version for the build requirement, so that new versions of Flit can make changes without breaking older packages. E.g. A future major version is likely to drop support for `[tool.flit.metadata]` tables, in favour of the standardised `[project]` metadata table.